src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -2019,5 +2019,6 @@
   "com.minecolonies.coremod.gui.miner.levels": "Levels",
   "com.minecolonies.coremod.gui.workerhuts.miner": "Mine",
   "com.minecolonies.coremod.gui.workerhuts.miner.repair.enqueued": "Added the repair request to the queue of the miner. Miner will pick it up once the current task is finished.",
-  "com.minecolonies.coremod.gui.townhall.picktexturestyle": "Pick Citizen Style"
+  "com.minecolonies.coremod.gui.townhall.picktexturestyle": "Pick Citizen Style",
+  "com.minecolonies.core.townhall.patreon": "Feature Limited to Supporters"
```

Comment:
Maybe change supporters to donators? It could be said that everyone who downloads the mod is a supporter of it, imo donator is more clear